app/Http/Controllers/Client/OrderController.php

```diff
@@ -79,4 +80,60 @@ public function removeProductInCart(Request $request)
             'status' => true,
         ]);
     }
+
+    /**
+     * Increase product quantity in order.
+     *
+     * @param  \App\Http\Requests\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function increaseQuantity(Request $request)
+    {
+        $productId = $request->product_id;
+        $this->orderService->increaseQuantity($productId);
+
+        return response()->json([
+            'status' => true,
+        ]);
+    }
+
+    /**
+     * Decrease product quantity in order.
+     *
+     * @param  \App\Http\Requests\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function decreaseQuantity(Request $request)
+    {
+        $productId = $request->product_id;
+
+        $this->orderService->decreaseQuantity($productId);
+
+        return response()->json([
+            'status' => true,
+        ]);
+    }
+
+    /**
+     * Checkout order and Store order data in database.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function checkout()
+    {
+        $currentUser = auth()->user();
+        $currentUserId = $currentUser->id;
+
+        $order = $currentUser->orders
```

Comment:
```suggestion
        $order = $currentUser->orders()
```
e thử sai như này xem được ko
làm như này là lọc xong r mới lấy ra
còn cách của e là lấy tất cả ra rồi mới lọc status = 2